### ISSUE TITLE
Quartz select dropdown white text on white background

### ISSUE THREAD

**Issue by skylarmt** [thomaspark/bootswatch#1118]
When using the Quartz theme, dropdown options are displaying with white text on a white background.

**Comment by skylarmt** [comment]
Adding this fixes it:

```css
select option {
    color: var(--bs-dark);
}
```

**Comment by thomaspark** [comment]
Hey @skylarmt, dropdowns should show up as white text on a translucent background, like so:

<img width="720" alt="Screen Shot 2021-07-17 at 3 50 40 PM" src="https://user-images.githubusercontent.com/900407/126048001-05c1c8d6-bb04-4da0-9621-9ff2ff1443d3.png">

Quartz is intended to be used with a dark background, and won't be readable with light backgrounds.

**Comment by skylarmt** [comment]
I'm referring to select dropdowns, like in a form, not dropdown menus:

```html
<select class="form-control">
    <option>This option is unreadable</option>
    <option>when the dropdown is open</option>
</select>
```

Looks like it works fine in Firefox but in Chromium-based browsers the options are unreadable white text on an opaque white background. This is reproducible on https://bootswatch.com/quartz/, see "Example select".  Tested with Chromium 91.0.4472.101 on Ubuntu 18.04, as well as with NW.js v0.53.0 (using Chromium 90.0.4430.93).

**Comment by catborise** [comment]
it is fixed but not 5.0.2 release. use latest github commits

**Comment by thomaspark** [comment]
Ah yeah, this was also reported in #1112. It's been fixed and will be included in the next release.
